src/components/ManualUploadModal.js

```diff
@@ -103,55 +115,58 @@ const ManualUploadModal = ({ isOpen, onClose }) => {
               ))}
             </Select>
           )}
-          {periodosOptions.length > 0 ? (
+
+          <Flex gap={4} my={4}>
```

Comment:
```suggestion
          <Flex gap={2} my={4}>
```

Creo que quedo un poco grande el gap, que pensas?